### PR TITLE
Update MainActivity.java

### PR DESCRIPTION
**Pull request by LeaHiHi**

### CODE REVIEW

**Review by LadyAuroralite — CHANGES_REQUESTED** [review]
good

File: app/src/main/java/com/instructure/template/projectCodeHere/layouts/MainActivity.java [hunk 1]
```diff
@@ -123,7 +124,13 @@ public void onResponse(@NotNull Call call, @NotNull Response response) {
...
                      Log.i(TAG, courses.toString());
                      for (GetCourses.CoursesResponse c : courses) {
-                         db.addDrawerItems(new PrimaryDrawerItem().withName(c.getName()));
+                         double tmp = 0;
+                         for (GetEnrollments.EnrollmentResponse i : enrollments) {
+                             if (i.getCourse_id()==c.getId()) {
+                                 tmp = i.getGrades().getCurrent_score();
+                             }
+                         }
+                         db.addDrawerItems(new PrimaryDrawerItem().withName(c.getName() + "\t\t%" + tmp));
```
Comment (HeyBanditoz) [inline app/src/main/java/com/instructure/template/projectCodeHere/layouts/MainActivity.java]: Swap percentages and the number around.
`db.addDrawerItems(new PrimaryDrawerItem().withName(c.getName() + "\t\t" + tmp + "%"));`

**Review by LadyAuroralite — COMMENTED** [review]
thanks

**Review by HeyBanditoz — CHANGES_REQUESTED** [review]

**Review by LadyAuroralite — APPROVED** [review]
your code is terrible but it gets the job done just like you except you dont get anything done

**Review by LadyAuroralite — APPROVED** [review]
ndfgbndfghsbdfghszdbfgz

**Review by HeyBanditoz — APPROVED** [review]
**_MAKE YOUR COMMIT NAMES MORE DESCRIPTIVE_**

**Pull request merged**